pubspec.yaml

```diff
@@ -9,11 +9,12 @@ environment:
 
 dependencies:
   cupertino_icons: ^1.0.3
+  device_preview: ^0.7.5
   flutter:
     sdk: flutter
   flutter_feather_icons: ^2.0.0+1
   google_fonts: ^2.1.0
-  provider: ^6.0.1
+  provider: ^5.0.0
```

Comment:
this version was required by `device_preview` ?

Reply:
It is

Reply:
https://github.com/aloisdeniel/flutter_device_preview/issues/140

Reply:
They have a pr for it they just need to merge it https://github.com/aloisdeniel/flutter_device_preview/pull/139

Reply:
got it merged